packages/teams-js/webpack.config.js

```diff
@@ -52,5 +64,34 @@ module.exports = {
       removeSource: true,
       outputAsModuleFolder: true,
     }),
+
+    // https://www.npmjs.com/package/webpack-subresource-integrity
+    new SubresourceIntegrityPlugin({ enabled: true }),
+
+    // Webpackmanifest produces the json file containing asset(JS file) and its corresponding hash values(Example: https://github.com/waysact/webpack-subresource-integrity/blob/main/examples/webpack-assets-manifest/webpack.config.js)
+    new WebpackAssetsManifest({
+      integrity: true,
+      integrityHashes: ['sha384'],
+      output: 'MicrosftTeams-manifest.json',
+    }),
+    {
+      apply: compiler => {
+        compiler.hooks.done.tap('wsi-test', () => {
+          const manifest = JSON.parse(readFileSync(join(__dirname, 'dist/MicrosftTeams-manifest.json'), 'utf-8'));
+          // If for some reason hash was not generated for the assets, this test will fail in build.
+          expect(manifest['MicrosoftTeams.min.js'].integrity).toMatch(/sha384-.*/);
+
+          // Updating the old hash value with the new one in all places.
+          let currentHashValue = 'sha384-RhShpWwRxSnc+keX0WZxCsf8olaKOOw416Jky+StW6qhctsNR3GMwav/hMB1Snas';
+          replace({
+            regex: currentHashValue,
+            replacement: manifest['MicrosoftTeams.min.js'].integrity,
+            paths: ['.'],
+            recursive: true,
+            silent: true,
```

Comment:
I don't think this is exactly what we want. All places we have HTML script tags with URLs and integrity hashes should always have a tuple that match, e.g., we don't want to have the prod CDN URL in our README file HTML script tag matched with an integrity hash that won't work with it.